### ISSUE TITLE
import all skymiles pages into sharepoint

### ISSUE THREAD

**Issue by ramboz** [hlxsites/delta#34]
List extracted from the sitemap:

https://www.delta.com/content/www/en_US/skymiles/elevate-your-status.html
https://www.delta.com/content/www/en_US/skymiles/how-to-use-miles/skymiles-experiences.html
https://www.delta.com/content/www/en_US/skymiles/medallion-program/reclaim-my-status.html
https://www.delta.com/us/en/skymiles/airline-credit-cards/american-express-business-cards
https://www.delta.com/us/en/skymiles/airline-credit-cards/american-express-personal-cards
https://www.delta.com/us/en/skymiles/airline-credit-cards/overview
https://www.delta.com/us/en/skymiles/airline-credit-cards/international-credit-cards
https://www.delta.com/us/en/skymiles/airline-credit-cards/pay-with-miles
https://www.delta.com/us/en/skymiles/airline-credit-cards/pay-with-miles-international
https://www.delta.com/us/en/skymiles/airline-credit-cards/takeoff-15
https://www.delta.com/us/en/skymiles/how-to-earn-miles/skymiles-partners-offers
https://www.delta.com/us/en/skymiles/how-to-earn-miles/airline-partners
https://www.delta.com/us/en/skymiles/how-to-earn-miles/convert-partner-points
https://www.delta.com/us/en/skymiles/how-to-earn-miles/earn-with-delta
https://www.delta.com/us/en/skymiles/how-to-earn-miles/exception-fares
https://www.delta.com/us/en/skymiles/how-to-earn-miles/overview
https://www.delta.com/us/en/skymiles/how-to-earn-miles/vacations-and-cruises
https://www.delta.com/us/en/skymiles/how-to-use-miles/buy-gift-transfer-donate-miles
https://www.delta.com/us/en/skymiles/how-to-use-miles/overview
https://www.delta.com/us/en/skymiles/how-to-use-miles/shop-with-miles
https://www.delta.com/us/en/skymiles/how-to-use-miles/travel-with-miles
https://www.delta.com/us/en/skymiles/how-to-use-miles/upgrades-and-more
https://www.delta.com/us/en/skymiles/overview
https://www.delta.com/us/en/skymiles/medallion-program/diamond-medallion-status
https://www.delta.com/us/en/skymiles/medallion-program/elite-benefits-partners
https://www.delta.com/us/en/skymiles/medallion-program/gold-medallion-status
https://www.delta.com/us/en/skymiles/medallion-program/how-to-qualify
https://www.delta.com/us/en/skymiles/medallion-program/overview
https://www.delta.com/us/en/skymiles/medallion-program/international-partner-skyteam-benefits
https://www.delta.com/us/en/skymiles/medallion-program/medallion-benefits
https://www.delta.com/us/en/skymiles/medallion-program/medallion-upgrades
https://www.delta.com/us/en/skymiles/medallion-program/million-miler
https://www.delta.com/us/en/skymiles/medallion-program/platinum-medallion-status
https://www.delta.com/us/en/skymiles/medallion-program/silver-medallion-status
https://www.delta.com/us/en/skymiles/medallion-program/status-match-challenge
https://www.delta.com/us/en/skymiles/medallion-program/2023-skymiles-program-updates
https://www.delta.com/us/en/skymiles/program-resources/get-the-most
https://www.delta.com/us/en/skymiles/program-resources/overview
https://www.delta.com/us/en/skymiles/program-resources/news-and-updates
https://www.delta.com/us/en/skymiles/program-resources/program-rules
https://www.delta.com/us/en/skymiles/program-resources/ways-to-upgrade

Importer docs: https://www.hlx.live/developer/franklin-importer

**Comment by ramboz** [comment]
@kptdobe We wanted to look into importing the content for delta.com, but since it's a time-sensitive matter, @davidnuescheler suggested we direct ask you to look into it so we don't lose time broken imports that we might mess up ourselves 😁

**Comment by ramboz** [comment]
@davidnuescheler care to chime in here, since you recommended we ask the import team to handle this?

If it's just manual copy/pasting, I'm happy to take a look at it since this is already what we did for our reference page.

**Comment by kptdobe** [comment]
I sensed the urgency thus I have made a quick first import. All files have been created with raw content in it - less copy/paste required. Now only missing the blocks. All pages seem to be pretty different and have a collection of blocks to handle. Still, I have added handling for "Table" block (benefit table - https://www.delta.com/us/en/skymiles/medallion-program/medallion-benefits) and TOC block ("links on the page" - https://www.delta.com/us/en/skymiles/program-resources/program-rules).
You will find all import details in https://adobe.sharepoint.com/:x:/r/sites/HelixProjects/_layouts/15/Doc.aspx?sourcedoc=%7B9D99AB58-414D-4336-8F02-8D7AB8298280%7D&file=import-report.xlsx&action=default
Import code: https://github.com/hlxsites/delta/pull/55

If you find things that can be applied to all the files, I can easily add some more code to handle it and re-run the full import. But looking at most of the files, it will require to build the blocks for each case and content can be adjusted manually.
